back/engines/commercial/smart_groups/config/locales/en.yml

```diff
@@ -32,6 +32,11 @@ en:
       not_ends_on: "%{property} doesn't end on %{value}"
     email:
       property: "e-mail"
+    event_attendances:
+      attends_something: "User is registered for at least one event"
+      attends_nothing: "User is not registered for any event"
+      attends_some_of: "User is registered for at least one of the following events: %{value}"
```

Comment:
I wonder if these copies are not a bit long. Do you think they will be still be readable in the UI?
<img width="714" alt="Screenshot 2023-09-06 at 11 01 26" src="https://github.com/CitizenLabDotCo/citizenlab/assets/31925816/daf3616a-0822-4ef0-b6b4-c60dee62a34a">

Reply:
I think that's ok :)

<img width="698" alt="image" src="https://github.com/CitizenLabDotCo/citizenlab/assets/209268/08bd45c2-fbad-4db8-a097-65d9dc17eb21">

Also, those copies are not used in the smart-group wizard, but in the participation conditions returned by `WebApi::V1::PermissionsController#participation_conditions`.